internal/pkg/aws/codestar/codestar.go

```diff
@@ -19,31 +19,33 @@ type api interface {
 }
 
 // Connection represents a client to make requests to AWS CodeStarConnections.
-type Connection struct {
+type CodeStar struct {
 	client api
 }
 
 // New creates a new CloudFormation client.
-func New(s *session.Session) *Connection {
-	return &Connection{
+func New(s *session.Session) *CodeStar {
+	return &CodeStar{
 		codestarconnections.New(s),
 	}
 }
 
 // WaitUntilStatusAvailable blocks until the connection status has been updated from `PENDING` to `AVAILABLE` or until the max attempt window expires.
-func (c *Connection) WaitUntilStatusAvailable(ctx context.Context, connectionARN string) error {
+func (c *CodeStar) WaitUntilStatusAvailable(ctx context.Context, connectionARN string) error {
+	var interval time.Duration // Defaults to 0.
```

Comment:
Yup, exactly. Every five seconds up to 45 minutes.

Reply:
Is the idea that in case the connection is already instantly active we can try it? Otherwise I'm not really seeing the utility of using a var that changes from 0 to 5s like this (instead of just waiting 5s to try the first time).

Reply:
It was originally 10 seconds and PH suggested 5. Efe suggested trying immediately so that the tests don't have to wait for the sleep period to be over before running.

Reply:
I see, that makes sense! Carry on 🚀